src/ts-wave.ts

```diff
@@ -0,0 +1,103 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+
+type WaveDirection = 'top' | 'bottom';
+
+const DEVICE_VIEWBOX = {
+  mobile: '0 0 320 64',
+  tablet: '0 0 768 71',
+  desktop: '0 0 1920 122',
+} as const;
+
+type SvgDevice = keyof typeof DEVICE_VIEWBOX;
+
+@customElement('ts-wave')
+export class TsWave extends LitElement {
+  @property({ type: String })
+  direction: WaveDirection = 'top';
+
+  @state()
+  device: SvgDevice = 'desktop';
```

Comment:
chore: Detection of device should also happen initially and not only on resize.

I would expect on mobile on initial load, it will look like the desktop wave?